trade-finance-manager-ui/server/helpers/deal-cancellation-enabled.ts

```diff
@@ -0,0 +1,5 @@
+import { DEAL_SUBMISSION_TYPE, dealSubmissionType, isTfmDealCancellationFeatureFlagEnabled } from '@ukef/dtfs2-common';
+
+export const dealCancellationEnabled = (submissionType: dealSubmissionType): boolean => {
+  return isTfmDealCancellationFeatureFlagEnabled() && (submissionType === DEAL_SUBMISSION_TYPE.AIN || submissionType === DEAL_SUBMISSION_TYPE.MIN);
```

Comment:
Think its something to cover in a future ticket (once we've implemented cancelling!) but we'll want to add `deal.tfm.stage !== 'Cancelled` (or however we do this)

Reply:
Ooh yes good point. I can add a TODO with ticket number DTFS2-7298?